src/wp-includes/load.php

```diff
@@ -1532,6 +1535,9 @@ function wp_load_translations_early() {
 			$locales[] = $wp_local_package;
 		}
 
+		// Try the browser's locale
+		$locales = array_merge( $locales, get_locales_from_accept_language_header() );
```

Comment:
Should we do this only if `$locales` is empty?

Reply:
That might not be ideal. If a site has an outdated `WPLANG` constant for example, we'd still want to do this here.